### ISSUE TITLE
Cannot login with Google Chrome

### ISSUE THREAD

**Issue by atharvalele** [zachomedia/apple-music-webplayer#19]
Google Chrome v67.0.3396.99 (Official Build) (64-bit)
Manjaro Linux KDE

After signing into the auth window that pops up, the pop up window shows a blank white page. Closing that window does not do anything either. Neither does refreshing the page.

https://imgur.com/a/fvTZXEc

**Comment by zachomedia** [comment]
Can you check 2 things for me:

Any errors in the console of the Developer Tools?

Can you try with https://tools.applemusic.com?

1. Select a playlist
2. Click the … icon, click "open in a new window” and then click the sign-in button at the top right
3. Click “Play” on the playlist

**Comment by atharvalele** [comment]
I can't sign in on tools.applemusic.com either. The error in console is this

tools.applemusuic.com: https://paste.ubuntu.com/p/TtG3Q7Q6nf/
your website: https://paste.ubuntu.com/p/jmGYM8jyJC/

**Comment by zachomedia** [comment]
Thanks. Can you check if this is still happening?

This is very likely a problem on Apple's end and I don't really know if it's something I can debug.

**Comment by atharvalele** [comment]
Yup, it is still happening. On the apple website as well. Is there any way to open up a ticket / submit a bug report to them?

**Comment by zachomedia** [comment]
Try https://bugreport.apple.com (don't know if you need to be a developer to access it)

**Comment by atharvalele** [comment]
I've submitted a bug report here https://bugreport.apple.com/web/?problemID=41805849

Not sure if you can view it or it is my-account only. In any case, I'll keep you updated if Apple replies.

**Comment by zachomedia** [comment]
Thanks. I can't see the bug, but please let me know if/when you hear anything.

**Comment by atharvalele** [comment]
Huh, Apple just deleted my bug report for some reason without fixing it. I think I'll submit it again later.

**Comment by zachomedia** [comment]
@jaehess Can you look into what happened to @atharvalele's Radar? (41805849)

**Comment by jaehess** [comment]
@atharvalele Are you still having issues here?

**Comment by atharvalele** [comment]
> @atharvalele Are you still having issues here?

Yes, I am. Chrome v70.0.3538.77 (Official Build) (64-bit)

**Comment by JediBrooker** [comment]
Hey, I'm having this same issue. Any progress? Manjaro Gnome on Firefox. This seems like a good solution for us Linux users if we can get the login issue solved!
